README.md

```diff
@@ -1,31 +1,74 @@
++ Anytime I access this variable from this point forward in my program, it will be set to "Ian"
+```ruby
+name = "Ian"
+puts name # "Ian"
+```
++ We can change the value again by using the `=` sign
 
+```ruby
+name = "Ian"
+puts name # "Ian"
+name = "Ian Candy"
+puts name # "Ian Candy"
+```
++ Makes sense, right? So variables are great for things where, maybe when we're programming, we don't quite know what the value is going to be yet.
++ Let's use what we know about variables to write a program that tells you how old someone is in dog years. One hint - we usually say that your age in dog years is about 7 times what it is in regular years. How might we do this?
++ So first, let's set a variable called age. `age = 42`
++ Now, we can make another variable called `age_in_dog_years` and set it equal to age * 7 `age_in_dog_years = age * 7`
++ Lastly, we can print out our age in dog years: `puts age_in_dog_years`
++ So this works great, and we can change the age to a different value and it also works well. Something we'll look at in a future video is - how do we actually get this age from someplace else? Like, how do we have a user type in their age? That would make this even more interesting.
++ One last thing to note - there's a special kind of variable in Ruby called a Constant. A constant works just like any other variable, except it's value should stay the same throughout the program. Once it gets set, it will give us a warning if it ever changes.
++ In Ruby, you'll typically see this to call attention to important stuff that doesn't really change. In the example here, we're doing something called a `magic number` - if you didn't watch this video, you might be confused as to why the number 7 is here. So this is a good candidate to use a constant variable . Let's do that now:
++ I can define a constant the same way I do a variable, only it must start with a capital letter. By convention, for constant values like this, I'll use all caps.
+```ruby
+DOG_YEARS_PER_YEAR = 7
+```
++ Now, while running the program, if Ruby sees this constant change, it will throw a warning.
+```ruby
+DOG_YEARS_PER_YEAR = 7
+DOG_YEARS_PER_YEAR = 4 # Throws a warning
+```
++ So, the programmer can change this before it runs, but the value can't change mid-course so to speak.
+```ruby
+DOG_YEARS_PER_YEAR = 7
+age = 42
+age_in_dog_years = age * DOG_YEARS_PER_YEAR
+puts age_in_dog_years
+```
++ So that makes things a bit more clear. Now, if the programmer needs to change that constant, they can do so. For example, some people say 4 dog years per year is actually more accurate. We can easily change that if we need to, and it works well.
++ That's it for this one - to recap:
++ We talked about how expressions are used in programming, such as math expressions and variable assignment
++ We looked at the return values of different expressions
++ We described the Variable Lookup Expression, and how ruby finds a variable value
```

Comment:
The term "variable lookup" isn't really described here. This may be too technical unless you want to talk about it more in depth here. Instead, I would use the term "variable as an expression" and maybe just add a note that the formal name is "variable lookup expression".

Reply:
Yeah, let me try to simply this a bit.

Reply:
```suggestion
+ We described how ruby looks up a variables value, and why that's useful for our programs. 
```